spec/spec_helper.rb

```diff
@@ -8,7 +8,7 @@
 require "bundler"
 Bundler.setup
 
-ENV['RAILS'] = '4.1.9'
+ENV['RAILS'] = '4.2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.